scripts/xtensa-build-zephyr.py

```diff
@@ -210,6 +234,13 @@ def parse_args():
 			parser.print_help()
 			sys.exit(0)
 
+	if args.fw_naming == 'AVS':
+		if not args.use_platform_subdir:
+			args.use_platform_subdir=True
+			warnings.warn(f"Argument --fw-naming 'AVS' must go with using --use-platform-subdir. Already enable it automatically.")
```

Comment:
I think this warning should be omitted. No need to warn that the documented structure is used and no need to warn that platforms do not overwrite each other's firmware file.

Reply:
Hi @marc-hb , thanks for the feedback but I just leave the warnings there.
Because maybe it's not so good to change the build option silently without telling anything.

Hope current warnings are acceptable, they do not cost too much actually